content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -571,229 +569,44 @@ Push the schema with a custom schema location:
-  provider = "sqlite"
-  url      = "file:my-database.db"
-}
-```
+### <inlinecode>migrate resolve</inlinecode>
 
 #### Arguments
 
-The point to migrate back to can be defined in any of the following three ways:
-
-| Argument  | Required | Description                                                                             | Default |
-| --------- | -------- | --------------------------------------------------------------------------------------- | ------- |
-| decrement | No       | Specifies the number of backwards migrations to apply.                                  | 1       |
-| name      | No       | Specifies where to migrate back to using the name of the final migration to apply.      |
-| timestamp | No       | Specifies where to migrate back to using the timestamp of the final migration to apply. |
-
-#### Options
-
-Additionally, the following options modify the behavior of the `migrate down` command:
-
-| Option            | Required | Description                                                                                                                                         | Default                                     |
-| ----------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------- |
-| `--experimental`  | Yes      | Enables use of Experimental commands                                                                                                                |                                             |
-| `-p`, `--preview` | No       | Preview the migration that would be created without writing any changes to the filesystem.                                                          |                                             |
-| `--schema`        | No       | Specifies the path to the desired `schema.prisma` file to be processed instead of the default path. Both absolute and relative paths are supported. | `./schema.prisma`, `./prisma/schema.prisma` |
+| Argument   | Required | Description                                                                                                                                       | Default                                          |
+| :--------- | :------- | :------------------------------------------------------------------------------------------------------------------------------------------------ | :----------------------------------------------- |
+| `--applied` | No       | Specifies the path to the desired schema.prisma file to be processed instead of the default path. Both absolute and relative paths are supported. | `./schema.prisma`<br /> `./prisma/schema.prisma` |
+| `--rolledback` | No       | Specifies the path to the desired schema.prisma file to be processed instead of the default path. Both absolute and relative paths are supported. | `./schema.prisma`<br /> `./prisma/schema.prisma` |
```

Comment:
Description `Specifies the path to the desired schema.prisma file to be processed instead of the default path. Both absolute and relative paths are supported.` looks like a copy paste, should be more about this:
`npx @prisma/cli@dev migrate resolve -h`

```diff
Resolve issues with database migrations in deployment databases: 
- recover from failed migrations
- baseline databases when starting to use Prisma Migrate on existing databases
- reconcile hotfixes done manually on databases with your migration history

Run "prisma migrate status" to identify if you need to use resolve.

Read more about resolving migration history issues: https://pris.ly/d/migrate-resolve

WARNING Prisma's migration functionality is currently in Preview (https://pris.ly/d/preview).
When using any of the commands below you need to explicitly opt-in via the --preview-feature flag.
  
Usage

  $ prisma migrate resolve [options] --preview-feature
  
Options

    -h, --help   Display this help message
      --schema   Custom path to your Prisma schema
+     --applied   Record a specific migration as applied
+ --rolled-back   Record a specific migration as rolled back

Examples

  Update migrations table, recording a specific migration as applied 
  $ prisma migrate resolve --applied 20201231000000_add_users_table --preview-feature

  Update migrations table, recording a specific migration as rolled back
  $ prisma migrate resolve --rolled-back 20201231000000_add_users_table --preview-feature

  Specify a schema
  $ prisma migrate resolve --rolled-back 20201231000000_add_users_table --schema=./schema.prisma --preview-feature
```